VKapp/VKapp/Extensions/UIImageView+Extension.swift

```diff
@@ -0,0 +1,20 @@
+// UIImageView+Extension.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Extension for adding image from Internet
+extension UIImageView {
+    func loadImage(imageURL: String) {
+        if let url = URL(string: imageURL) {
+            URLSession.shared.dataTask(with: url) { [weak self] data, _, _ in
+                guard let self = self else { return }
+                DispatchQueue.main.async {
+                    guard let data = data,
+                          let image = UIImage(data: data) else { return }
+                    self.image = image
+                }
+            }.resume()
+        }
+    }
```

Comment:
```suggestion
    func loadImage(imageURL: String) {
                DispatchQueue.main.async {
                    guard let data = data,
                          let image = UIImage(data: data) else { return }
                    self.image = image
                }
    }
```
загрузкой должен заниматься Сервис